src/libraries/System.Data.Common/System.Data.Common.sln

```diff
@@ -38,36 +42,27 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ref", "ref", "{9A105AFB-AA36-46E1-BC68-6724F5644D0C}"
 EndProject
 Global
-	GlobalSection(NestedProjects) = preSolution
-		{F3DC1DF9-698A-4970-9A5E-AB946D6F763B} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{4C5DDB34-2F67-40F2-9D59-A3CEBAE35BE5} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{0FCF2BA6-A5B3-4E1C-A49A-424E77602313} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{0AE3ABFE-6284-4119-A5DC-1844E6AB8E02} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{C3F9A601-07D8-43A3-A3CA-7A002358A102} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{7AB121D2-0AAC-48E0-A834-6E220ECFEC4D} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{6E353933-8CBD-467D-8FF1-00133C3F2036} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{68D868F6-1311-47A2-9DDE-22B64B5FF85E} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{09129981-6B0D-44C7-9F0A-AD2426106332} = {2FBD1B5D-0E5B-4D8C-A228-48B428725D67}
-		{0BD22D2A-C12C-4641-8F12-73D21AAAFBA3} = {12B0E17A-6957-425E-98E1-5F245010893A}
-		{BEBD7B5B-9544-42EB-B878-F009560CAAF4} = {12B0E17A-6957-425E-98E1-5F245010893A}
-		{5305244F-C52B-4772-A13D-C034213F3833} = {9A105AFB-AA36-46E1-BC68-6724F5644D0C}
-		{F865C526-79F8-403B-B5E4-52A90ECE21E3} = {9A105AFB-AA36-46E1-BC68-6724F5644D0C}
-		{A404332F-CCAA-4E22-94C1-635F7F19176F} = {9A105AFB-AA36-46E1-BC68-6724F5644D0C}
-		{BF66C27C-D1B1-4F86-A1E1-EDC63584E818} = {9A105AFB-AA36-46E1-BC68-6724F5644D0C}
-		{649CA543-288D-4A8B-AED3-E470DC7FF866} = {9A105AFB-AA36-46E1-BC68-6724F5644D0C}
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\System.Private.CoreLib\src\System.Private.CoreLib.Shared.projitems*{f3dc1df9-698a-4970-9a5e-ab946d6f763b}*SharedItemsImports = 5
```

Comment:
Can you undo the changes in the sln file? These are generated by a script, so it is better if we try to keep them just updated with that script.